lapis/cli/simulate.py

```diff
@@ -88,34 +108,84 @@ def cli(ctx, seed, until, log_tcp, log_file, log_telegraf, calculation_efficienc
         click.Choice(list(storage_import_mapper.keys())),
     ),
     default=(None, None, None),
+    multiple=True,
+    help="Tuple of `(storage_file,storage_content_file,storage_type)`",
+)
+@click.option(
+    "--remote-throughput",
+    "remote_throughput",
+    type=float,
+    default=10,
```

Comment:
What is the background of selecting `10` as a default here? Please keep in mind that the standardised data type used inside of Lapis is bytes.

Reply:
Typo. In `custom_simulate.py` the default is given as `1.0`.